server/lomas_server/dp_queries/dummy_dataset.py

```diff
@@ -1,34 +1,35 @@
-import datetime
-import random
-
 import numpy as np
 import pandas as pd
 
 from lomas_server.admin_database.admin_database import AdminDatabase
 from lomas_server.constants import (
-    DEFAULT_NUMERICAL_MAX,
```

Comment:
did you also remove them from constants.py ?

Reply:
done